chalk-solve/src/clauses/builtin_traits.rs

```diff
@@ -1,9 +1,10 @@
 use super::{builder::ClauseBuilder, generalize};
 use crate::{Interner, RustIrDatabase, TraitRef, WellKnownTrait};
-use chalk_ir::{Substitution, Ty};
+use chalk_ir::{ProjectionTy, Substitution, Ty};
 
 mod clone;
 mod copy;
+mod fn_;
```

Comment:
Why `fn_` instead of `fn`?

Reply:
I thought it was preferrable to avoid needing to use a raw identifier (`r#fn`).

Reply:
Oh right. I liked the `fn_family` name from @nathanwhit's PR then. Can you change it to that?